Dockerfile.alpine

```diff
@@ -1,5 +1,8 @@
 FROM amd64/alpine:3.15 AS build
 
+ARG version
+ARG information_version
```

Comment:
Thinking of local builds - what happens when we don't provide these arguments on the command line when invoking docker build? Should we set default values for them?

Reply:
Yea I can set them to default to 3.0.0 and v3.0.0